Libraries/Alert/NativeAlertManager.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * Copyright 2004-present Facebook. All Rights Reserved.
+ *
+ * @flow strict-local
+ * @format
+ */
+
+'use strict';
+
+import type {TurboModule} from 'RCTExport';
+import * as TurboModuleRegistry from 'TurboModuleRegistry';
+
+type Button = {|
+  text?: string,
+  onPress?: () => void,
+  style?: string,
```

Comment:
Do you mind to add some comments and already preparing this types for when the codegen starts to support enums and unions, like it's done in [NativeDialogManagerAndroid](https://github.com/facebook/react-native/blob/3ec1119e08ee214306add2174f3be7725fbcebb0/Libraries/NativeModules/specs/NativeDialogManagerAndroid.js#L18|L34)?